python/ray/worker.py

```diff
@@ -66,6 +66,7 @@
 UTIL_WORKER_MODE = 5
 
 ERROR_KEY_PREFIX = b"Error:"
+FILTER_LOGS_BY_JOB = True
```

Comment:
Can we make this a property of `ray.worker.global_worker`?

We should probably avoid mutating global state outside the singleton. Also, this might make our unit test output really weird.

Reply:
sure